starter-code/apple pie/styles.css

```diff
@@ -0,0 +1,53 @@
+body {
+    
+}
+
+header {
+    display: relative;
+    top: 0;
+    left: 0;
+    width: cover;
+    
+    margin-block-end: 20px;
+    border-bottom: 2px solid grey;
+}
+
+.header {
+    background-image: url(images/apple-pie.jpg)
```

Comment:
Here, you need to add a height and width attribute to get it to take up more space